containers/app/Dockerfile

```diff
@@ -31,7 +31,7 @@ FROM python:3.12-slim as runtime
 WORKDIR /app
 
 ENV RUN_AS_DEVIN=false
-ENV USE_HOST_NETWORK=false
+ENV USE_HOST_NETWORK=true
```

Comment:
Is this a necessary part of this PR?

I'm going to see if it works for me on latest docker on mac

Reply:
revert!

Reply:
Hmm I'm still seeing `USE_HOST_NETWORK=true` here

Reply:
This is the results of https://github.com/OpenDevin/OpenDevin/pull/1426, which i guess will be the dependency for this one